packages/cozy-harvest-lib/src/components/KonnectorConfiguration/DataTab/index.jsx

```diff
@@ -18,7 +18,19 @@ import tabSpecs from '../tabSpecs'
 import { useTrackPage } from '../../../components/hoc/tracking'
 import RedirectToAccountFormButton from '../../RedirectToAccountFormButton'
 
-export const DataTab = ({ konnector, trigger, client, flow }) => {
+const findSuitableDataCards = (permissions, doctypeToDataCard) => {
+  return Object.values(permissions)
+    .map(permission => doctypeToDataCard[permission.type])
+    .filter(Boolean)
+}
```

Comment:
C'est un peu dommage de se baser sur les permissions pour afficher ou non un bloc de données. 

On devrait plutôt se baser sur le manifest du connecteur, et le manifest devrait déclarer les blocs qu'il veut afficher dans Harvest, un peu à la manière des champs de login. Parce que c'est pas parce que mon connecteur a une permissions sur les fichiers que j'ai envie qu'il affiche le bloc de fichiers, ça devrait être décorrélé. 

A garder en tête pour les futures itérations sur cette partie, j'arrive après la bataille ici.

Reply:
> On devrait plutôt se baser sur le manifest du connecteur, et le manifest devrait déclarer les blocs qu'il veut afficher dans Harvest,

Le problème dans ce cas là, c'est repasser sur tout les connecteurs déjà existant et mettre cette déclaration. Personnellement je pense que ça suffirait de ne prendre que la première permission qui match et optionnellement de pouvoir dire sur une permission que tu ne veux pas qu'elle soit affichée en tant que datacard. Ou de pouvoir spécifier en dur le visualiseur que tu veux pour tes données.

```
datacards: {
  'io.cozy.permissions': false,
  'io.cozy.files': 'FileDataCard2' // already existing in harvest
}
```

> Parce que c'est pas parce que mon connecteur a une permissions sur les fichiers que j'ai envie qu'il affiche le bloc de fichiers, ça devrait être décorrélé.

C'est à débattre, je vois pas très bien un usecase ou en tant que développeur de connecteur tu veux afficher un viewer différent.

Reply:
J'ai fait une issue : https://github.com/cozy/cozy-libs/issues/1275

Reply:
On en discutera en atelier, car y a déjà pas mal de travail fait sur ce sujet, ce sera l'occasion de fournir toutes les billes à tout le monde

Reply:
> C'est à débattre, je vois pas très bien un usecase ou en tant que développeur de connecteur tu veux afficher un viewer différent.

Pour info, sur un connecteur qui aurait une permission sur un docType mais sans pour autant une volonté d'afficher le viewer sur cette donnée, il suffit de prendre le cas des connecteurs de factures qui demandent la permission sur les données bancaires pour créer leur bills. 
Je ne veux pas avoir d'encart 'Mes données bancaires' même si le connecteur demande les permissions sur ce docType.